pandacommon/kafkapublisher/KafkaPublisher.py

```diff
@@ -7,8 +7,8 @@
 import json
 import hashlib
 from confluent_kafka import Producer
-from pandacommon.pandalogger.logger_utils import logger_utils
-import config
+from pandacommon.pandalogger import logger_utils
+from . import kafka_config as config
```

Comment:
Is kafka_config a config file? It is not a good idea to have such a file under site-packages.
I suggest you move the file to /etc/panda or $VIRTUAL_ENV/etc/panda and specify it in

https://github.com/PanDAWMS/panda-common/blob/master/templates/panda_common.cfg.rpmnew

Reply:
It's a config file required by the patched CERN's confulent kafka python library to work with Kerberos auth:
https://gitlab.cern.ch/db/kafka-clients-example/-/blob/master/python/confluent-kafka-python/kerberos/config.py

Reply:
It is not convenient to edit /opt/panda/lib/python3.6/site-packages/pandacommon/kafkapublisher/kafka_config.py
for each panda installation. My point is that you should put the config parameters in ($VIRTUAL_ENV)/etc/panda
and let kafka_config read those parameters to set BASEDIR etc dynamically.
You can put the parameters in panda_common.cfg or a python file as you like.

Also I guess kafka_config.py doen't work outside of CERN since cern.ch is hardcoded at
https://github.com/PanDAWMS/panda-common/blob/master/pandacommon/kafkapublisher/kafka_config.py#L15

Reply:
Indeed I will do it the way you are suggesting - for now, I am not sure if this config file will be required for an outside CERN Kafka installation... only if for example the SLAC Kafka service (is there such thing or will we need to run it oursleves?) will support ONLY kerberos auth and nothing else.